packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -38,4 +70,177 @@ Content-Type: application/octet-stream;
   @post
   @route("/mixed-parts")
   op basic(@header contentType: "multipart/form-data", @body body: MultiPartRequest): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+Expect request:
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+Content-Type: text/plain
+
+123
+--abcde12345
+Content-Disposition: form-data; name="address"
+Content-Type: application/json
+
+{
+  "city": "X"
+}
+--abcde12345
+Content-Disposition: form-data; name="profileImage"; filename="<any-name-is-ok>"
+Content-Type: application/octet-stream
+
+{…file content…}
+--abcde12345--
+Content-Disposition: form-data; name="previousAddresses"
+Content-Type: application/json
```

Comment:
Is the content type correct on the wire? Do we have any spec here?

Reply:
According to https://datatracker.ietf.org/doc/html/rfc7578#section-4.4, if this part is not file, content-type is not necessary. Now cadl-ranch doesn't have the competence to check it so it is up to per lanuage.

Reply:
Then could you update the summary to indicate that Content-Type is optional here?

Reply:
Updated scenario docs.  
`do you know if the content type as text/plain is also acceptable?`
cadl-ranch will ignore the content-type for non-file part, while I still advise to pass the right content-type.

Reply:
I am okay with current status and let's keep monitoring the real service's behavior.